docs/user-guide/cli-install-configure-zosmf.md

```diff
@@ -0,0 +1,33 @@
+| Maximum region size | To prevent exceeds maximum region size errors, ensure that you have a TSO maximum region size of at least 65536 KB for the z/OS system. |
+| User IDs | User IDs require a TSO segment (access) and an OMVS segment. During workflow processing and REST API requests, z/OSMF may start one or more TSO address spaces under the following job names: <ul><li>*userid*</li><li>*substr(userid, 1, 6)//CN* (Console)</li></ul> Example: ```(userid = USRMY01, USRMY0CN )```
+
+## Selecting and configuring your z/OSMF plug-ins
+
+| Plug-in        | Functionality | Task |
+| ----------- | ----------- | ----------- |
+| (Optional) Cloud Portal | The Cloud Portal plug-in lets you make software services available to marketplace consumers and it adds the Marketplace and Marketplace Administration tasks to the z/OSMF navigation tree. | For information about how to enable the plug-in, see [Cloud provisioning marketplace](https://www.ibm.com/docs/en/zos/2.5.0?topic=services-cloud-provisioning-marketplace) on the IBM Documentation. |
+| Configuration Assistant | The Configuration Assistant plug-in lets z/OSMF configure TCP/IP policy-based networking functions.<br/>For more information about the functionality that the plug-in provides, see [Configuration Assistant task overview](https://www.ibm.com/docs/en/zos/2.2.0?topic=tasks-configuration-assistant-task-overview) on the IBM Documentation. | For information about how to enable the plug-in, see Updating z/OS for the [Configuration Assistant plug-in](https://www.ibm.com/docs/en/zos/2.2.0?topic=ins-updating-zos-configuration-assistant-plug-in) on the IBM Documentation. |
+| ISPF | The ISPF plug-in lets z/OSMF access traditional ISPF applications.<br/>For information about the functionality that the plug-in provides, see [ISPF task overview](https://www.ibm.com/docs/en/zos/2.2.0?topic=tasks-ispf-task-overview) on the IBM Documentation. | For information about how to enable the plug-in, see [Updating z/OS for the ISPF service](https://www.ibm.com/docs/en/zos/2.5.0?topic=service-updating-zos-ispf) on the IBM Documentation. |
+| Workload Management | The Workload Management plug-in lets z/OSMF operate and manage workload management service definitions and policies.<br/>For information about the functionality that the plug-in provides, see [Workload Management task overview](https://www.ibm.com/docs/en/zos/2.2.0?topic=tasks-workload-management-task-overview) on the IBM Documentation. | For information about how to enable the plug-in, see Updating z/OS for the [Updating z/OS for the Workload Management service](hhttps://www.ibm.com/docs/en/zos/2.5.0?topic=service-updating-zos-workload-management) on the IBM Documentation. |
```

Comment:
@adam-wolfe @zFernand0 -- I couldn't five the v2.5 equivalent for the v2.2 article ("Workload Management task overview") in the IBM Docs. Could one of you help me find a replacement?

Reply:
Is it this?
https://www.ibm.com/docs/en/zos/2.5.0?topic=performance-workload-management-task

Reply:
verifying... 😋 

Same a sprevious comment, I think they did a better job with the overview in 2.5 compared to 2.2 😋